examples/esp32spi/minimqtt_certificate_esp32spi.py

```diff
@@ -10,14 +10,14 @@
 
 import adafruit_minimqtt.adafruit_minimqtt as MQTT
 
-### WiFi ###
+# Get WiFi details and MQTT keys, ensure these are setup in settings.toml
+ssid = getenv("CIRCUITPY_WIFI_SSID")
+password = getenv("CIRCUITPY_WIFI_PASSWORD")
+broker = getenv("broker")
+username = getenv("username")
+paswword = getenv("paswword")
```

Comment:
getenv not defined

Reply:
@tyeth good catch. Fixed.

Ruff didn't catch that, so that's strange (cc @FoamyGuy)